actions/techdocs-rewrite-relative-links/action.yaml

```diff
@@ -38,13 +38,13 @@ runs:
 
   steps:
     - name: Checkout
-      if: ${{ !inputs.skip-checkout }}
+      if: ${{ inputs.checkout }}
```

Comment:
Should this be `checkout-repo`?

Reply:
Or perhaps `checkout-action-repository`?

Reply:
Yep, even better 😁